src/map/entities/mobentity.cpp

```diff
@@ -724,6 +724,15 @@ void CMobEntity::OnMobSkillFinished(CMobSkillState& state, action_t& action)
             {
                 angle = 45.0f;
             }
+
+            // for mobs that do not turn to face the target
+            if (m_Behaviour & BEHAVIOUR_NO_TURN)
+            {
+                // need to always add the single main target as otherwise player can interrupt cone skill
+                // by spinning around the mob which should not be possible
+                PAI->TargetFind->findSingleTarget(PTarget, findFlags);
```

Comment:
This should be inside the conal centered on mob check, ya? (m_aoe == 6)

Reply:
The idea is that no conal move should be interrupted due to change in target position between readying and actual usage, thus this applies also to other conal moves.

Reply:
> This should be inside the conal centered on mob check, ya? (m_aoe == 6)

The idea is that no conal move should be interrupted due to change in target position between readying and actual usage, thus this applies also to other conal moves.